std/mime/multipart.ts

```diff
@@ -281,10 +281,10 @@ export class MultipartReader {
    * null value means parsing or writing to file was failed in some reason.
    * @param maxMemory maximum memory size to store file in memory. bytes. @default 1048576 (1MB)
```

Comment:
@ry `go` [uses `10MB`](https://golang.org/src/mime/multipart/formdata.go) as default in `readForm`. Yet the comment in the code says `1MB`, just want confirmation. I think  `1MB` it's a good value to keep in Buffer.

Reply:
We should definitely follow Go's example. 10MB sounds good to me.

Reply:
Done. Reverted back to `10MB` and updated comment.